en/frames_multicopter/holybro_x500V2_pixhawk5x.md

```diff
@@ -0,0 +1,211 @@
+# Holybro X500 V2 + Pixhawk5x Build
+
+This topic provides full instructions for building the kit and configuring PX4 using *QGroundControl*.
+
+## Key information
+
+- **Kit:** [Holybro X500 V2 ARF Kit](http://shop.holybro.com/x500-v2-kit_p1288.html)
+- **Frame:** [Holybro X500 V2 Frame Kit](https://shop.holybro.com/x500-v2-kit_p1288.html)
```

Comment:
Kit & Frame links are the same, do we need to repeat it?

Reply:
No, this should be removed.

Reply:
Just left the Kit.